### PR TITLE
Allow to selectively disable filters

### PR DESCRIPTION
**Pull request by AlexanderYastrebov**
Signed-off-by: Alexander Yastrebov <alexander.yastrebov@zalando.de>

### CODE REVIEW

**Comment by MustafaSaber** [comment]
👍

File: skipper.go [hunk 1]
```diff
@@ -318,6 +318,9 @@ type Options struct {
 	// DefaultFilters will be applied to all routes automatically.
 	DefaultFilters *eskip.DefaultFilters
 
+	// A list of filters unavailable for use
+	DisabledFilters []string
```
Comment (szuecs) [inline skipper.go:322]: Proper godoc starts with the name of the field. 
So: `DisabledFilters is a list of ...`

File: skipper.go [hunk 2]
```diff
@@ -1647,11 +1650,25 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
...
+	// create a filter registry with the builtin and
+	// custom filter specs registered excluding disabled
+	disabledFilters := make(map[string]struct{})
+	for _, name := range o.DisabledFilters {
+		disabledFilters[name] = struct{}{}
+	}
+
+	registry := make(filters.Registry)
+	registerFilter := func(spec filters.Spec) {
+		if _, disabled := disabledFilters[spec.Name()]; !disabled {
```
Comment (szuecs) [inline skipper.go:1662]: IMO: disabled -> ok, because this is a very generic code pattern and reads much better than 3x disabled.

**Comment by szuecs** [comment]
:+1:

**Comment by MustafaSaber** [comment]
👍

**Pull request merged**